source/future.tex

```diff
@@ -2477,6 +2594,30 @@
 }
 \end{codeblock}
 
+\rSec2[depr.atomics.volatile]{Volatile access}
+
+If an atomic specialization has one of the following overloads,
+then that overload is available when \tcode{atomic<T>::is_always_lock_free}
+is \tcode{false}:
+\begin{codeblock}
+void store(T desired, memory_order order = memory_order::seq_cst) volatile noexcept;
+T operator=(T desired) volatile noexcept;
+T load(memory_order order = memory_order::seq_cst) const volatile noexcept;
+operator T() const volatile noexcept;
+T exchange(T desired, memory_order order = memory_order::seq_cst) volatile noexcept;
+bool compare_exchange_weak(T& expected, T desired,
+                           memory_order success, memory_order failure) volatile noexcept;
+bool compare_exchange_strong(T& expected, T desired,
+                             memory_order success, memory_order failure) volatile noexcept;
+bool compare_exchange_weak(T& expected, T desired,
+                           memory_order order = memory_order::seq_cst) volatile noexcept;
+bool compare_exchange_strong(T& expected, T desired,
+                             memory_order order = memory_order::seq_cst) volatile noexcept;
+T fetch_@\placeholdernc{key}@(T operand, memory_order order = memory_order::seq_cst) volatile noexcept;
+T operator @\placeholdernc{op}@=(T operand) volatile noexcept;
+T* fetch_@\placeholdernc{key}@(ptrdiff_t operand, memory_order order = memory_order::seq_cst) volatile noexcept;
+\end{codeblock}
+
 \rSec2[depr.atomics.nonmembers]{Non-member functions}
 
 \indexlibraryglobal{atomic_init}%
```

Comment:
The second part of this change to [depr.atomics] is missing:
"The following non-member function is available when atomic<T>::is_always_lock_free is false:"

I realize atomic_init was already deprecated, so maybe change the wording to say "For the volatile overload of this function, ..."

Reply:
Here's the problem: The entirety of atomic_init is deprecated and the "volatile" part is double-deprecated. So the "If" reduces to "If true".